modules/core/src/arithm.cpp

```diff
@@ -2130,1013 +2130,4 @@ cvMaxS( const void* srcarr1, double value, void* dstarr )
 }
 
 
-
-namespace cv { namespace hal {
-
-//=======================================
-
-#if (ARITHM_USE_IPP == 1)
-static inline void fixSteps(int width, int height, size_t elemSize, size_t& step1, size_t& step2, size_t& step)
```

Comment:
This massive code migration to another file "arithm.dispatch.cpp" is not required.
"dispatch.cpp" file is not required to enable "dispatched" optimizations.

Please move this part of code back (with added changes).

Reply:
done, thanks